modules/ztp-applying-the-ran-policies-for-monitoring-cluster-activity.adoc

```diff
@@ -6,26 +6,29 @@
 [id="ztp-applying-the-ran-policies-for-monitoring-cluster-activity_{context}"]
 = Applying the RAN policies for monitoring cluster activity
```

Comment:
Don't see this module in the build. If it will be included, it needs to be unwrapped.